common/generated_libs/network_protocol/network_protocol.proto

```diff
@@ -122,9 +122,7 @@ message ControllerStatus {
 message VentParams {
   required VentMode mode = 1;
 
-  required uint32 percent_inspiratory_o2 = 2; // FiO2 - frac inspiratory oxygen
   required uint32 peep_cm_h2o = 3; // PEEP - positive end-expiratory pressure
```

Comment:
You can now refer to https://github.com/RespiraWorks/VentilatorSoftware/wiki/Ventilator-Glossary here :)

Reply:
Done!